### DISCUSSION TITLE
Priority support for some charting/plotting library

### DISCUSSION THREAD

**Discussion by pbk0** [flet-dev/flet#245] [General]
As python is mostly used in data analytics and numerical computation it would be nice to add some support for plotting/charting.
Maybe [syncfusion](https://flutter.syncfusion.com/#/cartesian-charts/chart-types/line/default-line-chart) or [google charts](https://pub.dev/packages/charts_flutter)

If it is too much to work on then at least provide a guide to integrate `flutter` libraries in `flet`

**Comment by FeodorFitsner** [comment]
Syncfusion looks nice, but their license is no go: https://pub.dev/packages/syncfusion_flutter_charts/license - maybe we'll implement it as a separate package in the future.

There is an open issue for charts support - we could continue discussion there: https://github.com/flet-dev/flet/issues/68

So, Google charts looks like an abandoned "internal" project. Candidate #1 is https://pub.dev/packages/flutter_echarts
And just yesterday I discovered "Graphic" library which looks nice too: https://github.com/entronad/graphic

**Reply by pbk0** [reply]
Thanks for the comments.

**Comment by mrkvn** [comment]
I'm actually looking for a way to plot charts/dashboards too. So currently, there's no way, like using plotly or matplotlib?

**Comment by clstaudt** [comment]
Charts are also a must-have for my use case. Many Python plotting libraries, like plotly and altair, output HTML/CSS/JS. If flet had a way to display web content - problem solved quickly?

**Comment by PierreFaraut** [comment]
Hi team!
Trying to get some updates on this topic? 
I have to say that what stops me as user of flet is the lack of chart / visualization tools. 
If we could have an idea of when this topic could be tackle it would be awesome :) 
What do you think? 
Pierre
